projects/core/components/alert/alert.style.less

```diff
@@ -18,14 +18,17 @@
     &:not(:last-child) {
         margin-bottom: 0 !important;
     }
+
+    &:not(:last-child) > .t-wrapper {
+        margin-bottom: 0.75rem;
```

Comment:
Now if we provide `auto 3rem 2rem auto` as `TUI_ALERT_POSITION` so that alerts are aligned to the bottom and try closing the last one – the animation is a bit broken.